src/openassetio-core/include/openassetio/hostApi/Manager.hpp

```diff
@@ -260,6 +260,13 @@ class OPENASSETIO_CORE_EXPORT Manager final {
    * @param terms Map of terms to be substituted by the manager.
    *
    * @return Substituted map of terms.
+   *
+   * @throws errors.NotImplementedException Thrown when this method is not
```

Comment:
[consider][minor] Here and other copy-pasted text - line wrapping.

Reply:
Dealt with here : https://github.com/OpenAssetIO/OpenAssetIO/pull/1141/commits/b13bc4c4e04c619dd07d4e539652a99ee5b20a0d